### PR TITLE
add support for jupyter lab

### PR DESCRIPTION
**Pull request by maartenbreddels**

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
Thanks for updating the cookiecutter!

**Comment by flekschas** [comment]
@maartenbreddels Is there something special I need to do to get it working in JupyterLab? I just tested the `example.HelloWorld()` and while it works in Jupyter Notebook (`v5.5.0`) it's broken in JupyterLab (`v0.32.1`). All I am seeing is `Error displaying widget`.

File: {{cookiecutter.github_project_name}}/js/lib/labplugin.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var {{ cookiecutter.npm_package_name }} = require('./index');
+var base = require('@jupyter-widgets/base');
+
+module.exports = {
+  id: '{{ cookiecutter.npm_package_name }}',
+  requires: [base.IJupyterWidgetRegistry],
+  activate: function(app, widgets) {
+      widgets.registerWidget({
+          name: '{{ cookiecutter.npm_package_name }}',
+          version: {{ cookiecutter.npm_package_name }}.version,
```
Comment (wolfv) [inline {{cookiecutter.github_project_name}}/js/lib/labplugin.js]: this doesn't really work well for names with `-` inside, which seems to be encouraged.

Comment (jasongrout) [reply {{cookiecutter.github_project_name}}/js/lib/labplugin.js]: Very good point. There's no reason that variable name can't just be `plugin` or something.

**Pull request merged**